modules/carto/src/layers/quadbin-tile-layer.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  CompositeLayer,
+  CompositeLayerProps,
+  Layer,
+  LayersList,
+  UpdateParameters,
+  DefaultProps
+} from '@deck.gl/core';
+import {QuadkeyLayer, QuadkeyLayerProps} from '@deck.gl/geo-layers';
+import QuadbinTileset2D, {quadbinToTile, tileToQuadkey} from './quadbin-tileset-2d';
+import SpatialIndexTileLayer from './spatial-index-tile-layer';
+
+const renderSubLayers = props => {
+  const {data} = props;
+  if (!data || !data.length) return null;
+  return new QuadkeyLayer(props, {
```

Comment:
I'm fine keeping the QuadkeyLayer in a first implementation, but as you suggested in the PR description we've room for optimization here

Reply:
The main drawback is that the user will receive a Quadkey and they're working with Quadbin, so agree would be better to have the QuadBinLayer and also save some conversions